### DISCUSSION TITLE
500 errors on fresh digital ocean install

### DISCUSSION THREAD

**Discussion by thedug** [mastodon/mastodon#21153] [General]
Hi,

I'm getting 500 errors after a fresh install on digital ocean using the mastodon droplet. 

Any tips on how to trouble shoot, I'm not seeing much to go on.  The log folder is empty and when I run journalctl I only get this back:

No journal files were found.
-- No entries --

**Comment by MaryMuse** [comment]
I just had this happen on mine. What it ended up being was my Amazon S3 settings not being correct. Also a full reboot of my droplet resolved the empty journalctl.

**Reply by thedug** [reply]
Oh.. wow. Yeah.. Rebooting brought the logs in.

The README should mention that.

My issue was that I didn't have "ACLs Enabled"  under Bucket / Permissions / Object Ownership 

I referenced a few really good tutorials and none of them mentioned this so I'm going circle back to those and let them know.
